powersimdata/input/transform_grid.py

```diff
@@ -356,32 +356,33 @@ def _add_gencost(self):
 
     def _add_storage(self):
         """Adds storage to the grid."""
-        storage_id = self.grid.plant.shape[0]
-        for bus_id, value in self.ct["storage"]["bus_id"].items():
-            storage_id += 1
-            self._add_storage_unit(bus_id, value)
+        first_storage_id = self.grid.plant.index.max() + 1
```

Comment:
I guess we could rename `first_storage_id` to `storage_id` and then do `storage_id += i` in the loop

Reply:
For some reason I didn't get the email about this message until after the merge, sorry.

Reply:
No worries. Very minor.